core/node/node_framework/src/task.rs

```diff
@@ -7,7 +7,7 @@ use crate::service::StopReceiver;
 
 /// A task implementation.
 #[async_trait::async_trait]
-pub trait Task: 'static + Send + Sync {
```

Comment:
why not `Sync` anymore?

Reply:
It doesn't have to be `Sync` in fact (no task is accessed from two thread simultaneously), so it was an incorrect bound.